docs/develop/support-ie-11.md

```diff
@@ -0,0 +1,50 @@
+
+You can write your code in either TypeScript or in a version of JavaScript later than ES5 and then transpile it at build-time into ES5 JavaScript. The resulting ES5 files are what you upload to your add-in's web application.
+
+There are two popular transpilers. Both of them can work with source files that are TypeScript or post-ES5 JavaScript. They will both also work with React files (.jsx and .tsx).
+
+- [babel](https://babeljs.io/)
+- [tsc](https://www.typescriptlang.org/index.html)
+
+See the documentation for either of them for information about installing and configuring the transpiler in your add-in project. We recommend that you use a task runner, such as [Grunt](https://gruntjs.com/) or [WebPack](https://webpack.js.org/) to automate the transpilation. For examples of add-ins that use transpilers, open [Office Add-ins Patterns and Practices](https://github.com/OfficeDev/PnP-OfficeAddins) repo and search for “target: es5” to find samples that use tsc and search for "babel" to find samples that use babel.
+
+> [!NOTE]
+> If you are using Visual Studio (not Visual Studio Code), tsc is probably easiest to use. You can install support for it with a nuget package. For more information, see [JavaScript and TypeScript in Visual Studio 2019](/visualstudio/javascript/javascript-in-vs-2019). To use babel with Visual Studio, you could create a build script and/or use the Task Runner Explorer in Visual Studio and tools like the [WebPack Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.WebPackTaskRunner) or [NPM Task Runner](https://marketplace.visualstudio.com/items?itemName=MadsKristensen.NPMTaskRunner).
+
+## Use a polyfill
+
+A [polyfill](https://en.wikipedia.org/wiki/Polyfill_(programming)) is JavaScript that duplicates some functionality that was introduced in a relatively recent version of JavaScript, but the polyfill is written in an earlier version of JavaScript; so it can run in browsers that don't support the later version. For example, the string method `startsWith` wasn't part of the ES5 version of JavaScript, and so it won't run in Internet Explorer 11; but there are polyfill libraries, written in ES5, that define and implement a `startsWith` method. We recommend the [core-js](https://github.com/zloirock/core-js) polyfill library.
+
+To use a polyfill library you simply load it like any other JavaScript file or module. For example, you can use a `<script>` tag in the add-in's home page HTML file (for example `<script src="/js/core-js.js"></script>`), or you can use an `import` statement in a JavaScript file (for example, `import 'core-js';`). When the JavaScript engine sees a method like `startsWith`, it will first look to see if there is a method of that name built into the language. If there is, it will call the native method. If, and only if, the method isn't built-in, the engine will look in all loaded files for it. So, the polyfilled version isn't used in browsers that support the native version.
```

Comment:
```suggestion
To use a polyfill library, load it like any other JavaScript file or module. For example, you can use a `<script>` tag in the add-in's home page HTML file (for example `<script src="/js/core-js.js"></script>`), or you can use an `import` statement in a JavaScript file (for example, `import 'core-js';`). When the JavaScript engine sees a method like `startsWith`, it will first look to see if there is a method of that name built into the language. If there is, it will call the native method. If, and only if, the method isn't built-in, the engine will look in all loaded files for it. So, the polyfilled version isn't used in browsers that support the native version.
```